testkit/src/main/resources/reference.conf

```diff
@@ -18,7 +18,7 @@ akka.kafka.testkit.testcontainers {
   # define this to select a different Kafka version by choosing the desired version of Confluent Platform
   # available Docker images: https://hub.docker.com/r/confluentinc/cp-kafka/tags
   # Kafka versions in Confluent Platform: https://docs.confluent.io/current/installation/versions-interoperability.html
-  confluent-platform-version = "6.0.0"
+  confluent-platform-version = "6.0.1"
```

Comment:
Should we put the image name into config as well?

Reply:
Good idea.